msibi/state.py

```diff
@@ -33,9 +51,9 @@ class State(object):
         True if each query trajectory is backed up (default=False)
 
     """
-    def __init__(self, k, T, state_dir='', traj_file=None, top_file=None,
+    def __init__(self, kT, state_dir='', traj_file=None, top_file=None,
                  name=None, backup_trajectory=False):
-        self.kT = k * T
+        self.kT = kT
```

Comment:
Ok so do we just want to completely drop 1.0 support then? Or are we going to divide the user input by `k` and then passing that along for HOOMD 1.0 users?